lib/foreman/telemetry.rb

```diff
@@ -96,6 +96,16 @@ def enabled?
       @sinks.count > 1
     end
 
+    def allowed_tags(labels)
```

Comment:
We would need to allow `nil` here, so what renaming it and then using the variable?
```suggestion
    def initialize_allowed_tags(labels)
```